content/influxdb/v2.4/reference/cli/influx/auth/create.md

```diff
@@ -11,6 +11,64 @@ updated_in: CLI v2.3.0
 
 The `influx auth create` command creates an API token in InfluxDB.
 
+
+{{% warn %}}
+InfluxDB 2.4 introduced a bug that prevents you from creating an **all-access** or **operator** token using the `influx auth create` command, and causes the following error: `Error: could not write auth with provided arguments: 403 Forbidden: permission.`
+
+Until this bug is resolved in the next influx CLI release, please use the [workaround below to create an all-access or operator token](/influxdb/v2.4/security/tokens/create-token/#workaround-to-create-an-all-access-or-operator-token).
+{{% /warn %}}
+
+### **Workaround:** To create an all-access or operator token
+
+- Use the following command to create an [all-access](/influxdb/v2.4/security/tokens/#all-access-token) or [operator](/influxdb/v2.4/security/tokens/#operator-token) token. For an **operator** token, you must also include the `--read-orgs` and `--write-orgs` flags.
```

Comment:
```suggestion
Use the following command to create an [all-access](/influxdb/v2.4/security/tokens/#all-access-token) or [operator](/influxdb/v2.4/security/tokens/#operator-token) token. For an **operator** token, you must also include the `--read-orgs` and `--write-orgs` flags.
```

Reply:
Thank you, @sanderson--added bullet per [style here](https://developers.google.com/style/procedures#single-step-procedures) and bc I think it looks more crisp :-)